### DISCUSSION TITLE
Select Parameter Suggestions

### DISCUSSION THREAD

**Discussion by RagingRowen** [ikemen-engine/Ikemen-GO#906] [Feature Request] [answered]
**suborder**: 
The match the character/stage will appear in based on the order chosen. 
Example: 
The Tournament's Final Arena (ala CvS2) being selected for the final match for order=1.
Usage: 
order=1, suborder=8

**group**: 
If an opponent in arcade mode is in a group, other fighters from that group will be alongside them if it's a Team.
Example: Teams in KOF Arcade Modes.
Usage: 
Kyo, group=Japan
Benimaru, group=Japan
Ryu, group=Ansatsuken
Ken, group=Ansatsuken

**Comment by K4thos** [comment, marked as answer]
Arcade paths feature can be used to implement all the stuff you've proposed (and more, thanks to its flexibility), without need for extra parameters
https://github.com/ikemen-engine/Ikemen-GO/wiki/Miscellaneous-Info/#arcs
